### PR TITLE
Refactor FallingBalls MPP sample to follow KMM project structure.

### PR DESCRIPTION
**Pull request by pjBooms**
As a result iOS target can be run and debugged using KMM IDE tooling.

### CODE REVIEW

File: experimental/examples/falling-balls-mpp/README.md [hunk 1]
```diff
@@ -1,14 +1,34 @@
-## Run native on MacOS
-  `./gradlew runDebugExecutableMacosX64` (Works on Intel processors)
+# Falling Balls game
+
+Game can run on Android, iOS, desktop or in a browser.
+
+*Prerequisites*: to run on iOS and Android, you should have "Kotlin Multiplatform Mobile" plugin installed either 
+                 in Android Studio or in Intellij IDEA Ultimate or in AppCode with configured cocoapods.
```
Comment (dima-avdeev-jb) [inline experimental/examples/falling-balls-mpp/README.md:6]:
```suggestion
                 in Android Studio or in AppCode with configured cocoapods.
```

**Comment by dima-avdeev-jb** [comment]
We can add to .gitignore:
*.xcworkspace
Podfile.lock
Pods

File: experimental/examples/falling-balls-mpp/build.gradle.kts [hunk 2]
```diff
@@ -1,225 +1,22 @@
...
-
-    defaultConfig {
-        minSdk = 26
-        targetSdk = 32
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        mavenCentral()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
```
Comment (dima-avdeev-jb) [inline experimental/examples/falling-balls-mpp/build.gradle.kts:20]: also may be  add: mavenLocal()

File: experimental/examples/falling-balls-mpp/settings.gradle.kts [hunk 3]
```diff
@@ -1,23 +1,15 @@
 pluginManagement {
     repositories {
-        mavenLocal()
-        mavenCentral()
-        gradlePluginPortal()
-        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
         google()
+        jcenter()
+        gradlePluginPortal()
+        mavenCentral()
```
Comment (dima-avdeev-jb) [inline experimental/examples/falling-balls-mpp/settings.gradle.kts:6]: also mavenLocal() for internal testing

**Review by dima-avdeev-jb — APPROVED** [review]

File: experimental/examples/falling-balls-mpp/build.gradle.kts (hunk 2 above)
Comment (pjBooms) [reply experimental/examples/falling-balls-mpp/build.gradle.kts:22]: done

File: experimental/examples/falling-balls-mpp/README.md (hunk 1 above)
Comment (pjBooms) [reply experimental/examples/falling-balls-mpp/README.md]: done

File: experimental/examples/falling-balls-mpp/settings.gradle.kts (hunk 3 above)
Comment (pjBooms) [reply experimental/examples/falling-balls-mpp/settings.gradle.kts:6]: done

**Pull request merged**